docs/source/developers/ROADMAP_0_4.md

```diff
@@ -0,0 +1,90 @@
+# Roadmap
+
+## For 0.4.* series of releases - November 2020
```

Comment:
January 2021?

Reply:
we'll strictly speaking the first 0.4 release was back then and the doc started in january. i think it's ok. we probably shouldn't let it sit as long next time, but i'd probably leave as is now

Reply:
eh, still feels like backdating a contract, but 🤷. Happy to leave it be.